hack/test

```diff
@@ -90,7 +90,7 @@ fi
 DOCKERFILE_RELEASES_CUSTOM=""
 if [ "$TEST_DOCKERFILE" == 1 ]; then
   if [ -z $DOCKERFILE_RELEASES ]; then
-    DOCKERFILE_RELEASES="mainline experimental mounts secrets ssh"
```

Comment:
should we add `mounts` , `secrets`, and `ssh` as aliases for the `stable` image, so that dockerfiles that refer that syntax will get the current stable, and not some outdated version?

Reply:
Or perhaps also _ignore_ the `syntax` for current versions as it would now be the default, so don't (try to ) pull the frontend?

Reply:
As long as DockerHub doesn't GC the latest `mounts`, `secrets`, and `ssh` images, I think it's fine? I think if secrets suddenly is supported on the `mounts` image that would be more confusing as an user.

Reply:
There is no `mounts`, `secrets` channel in dockerhub. Only daily builds that in `dockerfile-upstream` are timestamped already.